main_test.go

```diff
@@ -12,13 +12,13 @@ import (
 	"github.com/jackc/pgx/v5"
 	"github.com/jackc/pgx/v5/pgconn"
 	"github.com/jackc/pgx/v5/pgtype"
+	pgcapturecursor "github.com/replicase/pgcapture/pkg/cursor"
 	replicasecursor "github.com/replicase/pgcapture/pkg/cursor"
 	replicasepb "github.com/replicase/pgcapture/pkg/pb"
+	pgcapturesink "github.com/replicase/pgcapture/pkg/sink"
 	replicasesink "github.com/replicase/pgcapture/pkg/sink"
+	pgcapturesource "github.com/replicase/pgcapture/pkg/source"
 	replicasesource "github.com/replicase/pgcapture/pkg/source"
-	pgcapturecursor "github.com/rueian/pgcapture/pkg/cursor"
-	pgcapturesink "github.com/rueian/pgcapture/pkg/sink"
-	pgcapturesource "github.com/rueian/pgcapture/pkg/source"
```

Comment:
We need to keep this, so we can benchmark default and pipeline mode.

Reply:
OK. Updated to only change the pkg name and drop the replace directive.